### ISSUE TITLE
Time calibration file not being produced by onsite calibration scripts

### ISSUE THREAD

**Issue by morcuended** [cta-observatory/cta-lstchain#260]
Hi @FrancaCassol 

I'm trying to generate the `time_calibration.Run{run}.0000.hdf5` file with `onsite_create_calibration_file.py`. However, the last command is not executed since in my installation there is no `lstchain_data_create_time_calibration_file` executable script. Is this executable generated when installing lstchain with `pip install -e .`?

Edit: I tried it with a fresh lstchain installation.

**Comment by FrancaCassol** [comment]
Hi @morcuended,

yes, you should have the master of cta-lstchain and install it like you indicate. The script is in 
lstchain/scripts
But at the moment there are some problems with run 1784. I could not get the time calibration file, I asked @pawel21  to have a look.

**Comment by morcuended** [comment]
In my case, I produced drs4 pedestal file with r1785 with no problems. Afterward tried to use this pedestal file to produce either the calibration and time_calibration files with r1786 but only the first one was produced. My guess is that the executable script is somehow not built or linked to the bin directory when creating the environment or installing lstchain whit pip. The only executable scripts that appear after the installation are:

```
(lstchain_v3_fresh) [daniel.morcuende@cp02 cta-lstchain]$ lstchain_data_create_
lstchain_data_create_calibration_file    lstchain_data_create_drs4_pedestal_file  lstchain_data_create_pedestal_file
```

**Comment by rlopezcoto** [comment]
Apparently this line was added in #248:
https://github.com/cta-observatory/cta-lstchain/blob/a9de4c33aee45af6a9f5e5c34fdd36e1d41dc694/lstchain/scripts/onsite/onsite_create_calibration_file.py#L116

but the script was not... could you please have a look @FrancaCassol? thanks

**Comment by FrancaCassol** [comment]
@morcuended, 
do you have the script in the lstchain/scripts directory?

**Comment by morcuended** [comment]
No. In lstchain/scripts/ there are:

```
(base) [daniel.morcuende@cp02 cta-lstchain]$ cd lstchain/scripts/
__init__.py                                 lstchain_data_r0_to_dl1.py                  lstchain_mc_r0_to_dl2.py                    lstchain_merge_hdf5_files.py
lstchain_data_create_drs4_pedestal_file.py  lstchain_mc_dl1ab.py                        lstchain_mc_rfperformance.py                lstchain_muon_reconstruction.py
lstchain_data_muon_analysis_dl1.py          lstchain_mc_dl1_to_dl2.py                   lstchain_mc_sensitivity.py                  onsite/
lstchain_data_muon_analysis.py              lstchain_mc_r0_to_dl1.py                    lstchain_mc_trainpipe.py                    tests/
```

**Comment by FrancaCassol** [comment]
in this case you need to fetch and merge the master of cta-lstchain

**Comment by rlopezcoto** [comment]
@FrancaCassol: we do not have that script in lstchain master:
https://github.com/cta-observatory/cta-lstchain/tree/master/lstchain/scripts

**Comment by morcuended** [comment]
What puzzle me now is that there is no `lstchain_data_create_calibration_file.py` but `lstchain_data_create_calibration_file` executable script is there in 
`~/.local/miniconda3/envs/lstchain_v3_fresh/bin/lstchain_data_create_calibration_file `

**Comment by FrancaCassol** [comment]
> @FrancaCassol: we do not have that script in lstchain master:
> https://github.com/cta-observatory/cta-lstchain/tree/master/lstchain/scripts

ops, perhaps I forgot to add it in the PR? I will add it soon (tI must leave now)

**Comment by rlopezcoto** [comment]
Solved by #261
